src/routes/index.js

```diff
@@ -38,11 +38,13 @@ app.get('/', (request, response) => {
 
 app.get('/complete', (request, response) => {
   getCompletedTodos()
-  .then((task) => {
-    response.render('completed', { task })
+  .then((tasks) => {
+    // console.log('BOB :: =>', tasks)
+    response.render('completed', { tasks })
   })
   .catch((err) => {
-
+    console.log(err.message);
+    response.json(err)
```

Comment:
I would make this two lines 
```
console.error
response.status(500)
```

Reply:
The reason is that to use the `err` object you would need to call it in your function like:
`app.get('/complete', (err, request, response, next){`